data/50-default.json

```diff
@@ -122,6 +122,18 @@
         "architecture": ["x86_64"]
     }
   },
+  {
+    "action": "install",
+    "serial": 2019120900,
```

Comment:
I believe, but am not sure, that this file should be kept in order of serial.

Reply:
Yeah, fixed that.